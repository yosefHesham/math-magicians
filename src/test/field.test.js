import React from 'react';
import renderer from 'react-test-renderer';
import Field from "../components/field"
describe('Test for home page', () => {
  test('Quote page should render correctly', () => {
    const field = renderer
      .create(<Field/>).toJSON()
      
    expect(field).toMatchSnapshot()
  });
});