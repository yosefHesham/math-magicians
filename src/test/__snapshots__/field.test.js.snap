// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for home page Quote page should render correctly 1`] = `
<div
  className="field   "
  id="field"
  onClick={[Function]}
  onKeyDown={[Function]}
  role="button"
  tabIndex={0}
/>
`;
